### ISSUE TITLE
Batch normalization is commented out - mistake?

### ISSUE THREAD

**Issue by oriabra** [kuleshov/audio-super-res#17]
Hi,  Thanks for submitting this paper, its super cool. 

I came across the lines regards applying batch normalization which are commented out (like on audionet.py:65 - # x = BatchNormalization(mode=2)(x)). On the same time on the paper you mentioned the batch normalization as part of the network architecture. 
The comment out is a mistake?

Thanks

**Comment by Sawyerb** [comment]
Hi @oriabra. This is not a mistake. (Although that line should have removed.) In the `tfilm` model, the T-FiLM layers provide the normalization.  The `audiounet` model is the version of our model without T-FiLM layers, which is one of the models that we used to benchmark our performance.
